internal/armada/server/lease_test.go

```diff
@@ -199,15 +209,15 @@ func (repo *mockJobRepository) ReturnLease(clusterId string, jobId string) (retu
 	return nil, nil
 }
 
-func (repo *mockJobRepository) DeleteJobs(jobs []*api.Job) map[*api.Job]error {
+func (repo *mockJobRepository) DeleteJobs(jobs []*api.Job) (map[*api.Job]error, error) {
```

Comment:
Was this changed to reflect the interface of DeleteJobs changing?

Reply:
Yes